actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -22,6 +23,8 @@ def initialize
         @disable_request_forgery_protection = false
         @allow_same_origin_as_host = true
         @filter_parameters = []
+
+        @health_check_application = ->(env) { Rails::HealthController.action(:show).call(env) }
```

Comment:
I wrapped the controller in a lambda to prevent having  to require is from action_cable (creating coupling between the railties and Action Cable) 
When action cable is loaded in the context of a rails app (including in standalone mode) the controller is in the autoloader path, it is not the case when Action Cable is used in isolation. In test for example.

Reply:
As you mention, this will create coupling between Action Cable and Railties, so I don't think it should be implemented this way. The usual route for something like this would be to provide a non-Railties dependent implementation as the default value for the config, and then overriding it with the Railties specific version in `ActionCable::Engine` (for example, see how `Rails.logger` gets set for `ActionCable::Server::Base.config`).

Reply:
Oh, good idea! 
How do you feel about this for the default?  

`[204, { "Content-Type" => "text/html", "date" => Time.now.httpdate }, []]` 

I already made the change if you feel like it's a sensible default.